controllers/org_rbac_controller.go

```diff
@@ -33,6 +34,10 @@ type OrganizationRBACReconciler struct {
 // In that case the controller will update it to bind to the organization.
 const LabelRoleBindingUninitialized = "appuio.io/uninitialized"
 
+// LabelNsNoAccess is used to speficy if RBAC rules should be created for a namespace.
+// If not specified it defaults to `admin` privileges on the namespace owned by the organization
+const LabelNsNoAccess = "appuio.io/no-access"
```

Comment:
I would prefer this to be a bit more meaningful.

```suggestion
const LabelNsNoAccess = "appuio.io/omit-default-rbac"
```

Alternatives `omit-organization-rbac`, `omit-rbac-creation`, `no-rbac-creation`, `no-user-access`, `no-access-management`, `no-rbac-magement`, `unmanaged-rbac`, `rbac-unmanaged`.

Reply:
Changed it to `appuio.io/no-rbac-creation`. Hope taht is ok for you.